File: QuickApp.Pro/ViewModels/SalesViews/SalesQuoteView.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace QuickApp.Pro.ViewModels.SalesViews
{
    public class SalesQuoteView
    {
        public SalesOrderQuoteView SalesOrderQuote { get; set; }

        public List<SalesOrderQuoteApproverListView> ApproverList { get; set; }

        public List<SalesOrderQuotePartView> Parts { get; set; }

        #region Data Source
        public IEnumerable<PriorityViewModel> Priorities
        {
            get; set;
        }

        public IEnumerable<MasterSalesQuoteTypesViewModel> SalesQuoteTypes
        {
            get; set;
        }

        public IEnumerable<CustomerTypeViewModel> CustomerTypes
        {
            get; set;
        }

        public IEnumerable<MasterSalesCreditTermsViewModel> CreditTerms
        {
            get; set;
        }

        public IEnumerable<MasterSalesProbablityViewModel> SalesProbablity
        {
            get; set;
        }

        public IEnumerable<MasterSalesLeadSourcesViewModel> LeadSources
        {
            get; set;
        }
        #endregion 
    }
}

File: QuickApp.Pro/ClientApp/app/models/sales/SalesQuoteView.ts
import { ISalesOrderQuote } from './ISalesOrderQuote';
import { ISalesOrderQuoteApproverList } from './ISalesOrderQuoteApproverList';
import { ISalesQuoteView } from './ISalesQuoteView';
import { ISalesOrderQuotePart } from './ISalesOrderQuotePart';
import { IPriority } from './IPriority';
import { ISalesQuoteType } from './ISalesQuoteType';
import { ICustomerType } from './ICustomerType';
import { ICreditTerm } from './ICreditTerm';
import { ISalesProbablity } from './ISalesProbablity';
import { ILeadSource } from './ILeadSource';


export class SalesQuoteView implements ISalesQuoteView {
    salesOrderQuote: ISalesOrderQuote;
    approverList: ISalesOrderQuoteApproverList[];
    parts: ISalesOrderQuotePart[]
    priorities: IPriority[];
    salesQuoteTypes: ISalesQuoteType[];
    customerTypes: ICustomerType[];
    creditTerms: ICreditTerm[];
    salesProbablity: ISalesProbablity[];
    leadSources: ILeadSource[];
}